src/RA_AchievementOverlay.h

```diff
@@ -4,41 +4,9 @@
 
 #include "RA_Achievement.h"
 #include "RA_User.h"
-#include "RA_Core.h"
-#include "RA_Interface.h"
+#include "RA_Core.h" // RA_Interface
 
-#include "services\ImageRepository.h"
-
-#include <map>
-#include <string>
-
-enum OverlayPage
-{
-    OP_ACHIEVEMENTS,
-    OP_FRIENDS,
-    OP_MESSAGES,
-    OP_NEWS,
-    OP_LEADERBOARDS,
-
-    OP_ACHIEVEMENT_EXAMINE,
-    OP_ACHIEVEMENT_COMPARE,
-    OP_FRIEND_EXAMINE,
-    OP_FRIEND_ADD,
-    OP_LEADERBOARD_EXAMINE,
-    OP_MESSAGE_VIEWER,
-
-    NumOverlayPages
-};
-
-enum TransitionState
-{
-    TS_OFF = 0,
-    TS_IN,
-    TS_HOLD,
-    TS_OUT,
-
-    TS__MAX
-};
+#include "services\ImageRepository.h" // map, string
```

Comment:
For source files it doesn't matter that much, but includes should be limited in headers. The comments are to show the removed includes are already included.